<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrescriptionDialog</class>
 <widget class="QDialog" name="PrescriptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>处方信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_record">
       <property name="text">
        <string>就诊记录ID*:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="medicalRecordIdSpin">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_medicine">
       <property name="text">
        <string>药品ID*:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="medicineIdSpin">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_quantity">
       <property name="text">
        <string>数量*:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="quantitySpin">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_dosage">
       <property name="text">
        <string>用法:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="dosageEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_frequency">
       <property name="text">
        <string>频次:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="frequencyEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_duration">
       <property name="text">
        <string>疗程:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="durationEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_instructions">
       <property name="text">
        <string>说明:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QTextEdit" name="instructionsEdit">
       <property name="maximumHeight">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrescriptionDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrescriptionDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
